docs/configuration.md

```diff
@@ -52,8 +52,10 @@ metadata:
 spec:
   api:
     address: 192.168.68.104
+    onlyBindToAddress: false
```

Comment:
As it defaults to `false` and it's `omitempty`, the `k0s kubeconfig create` command won't include that field.

```suggestion
```